CHANGELOG.md

```diff
@@ -1,223 +1,225 @@
 # Changelog
 
+# Historical Manual Changelog
+
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
-
-## [1.14.0] - 2020-05-26
+## [v1.14.0] (2020-05-26)
 ### Added
-- ip: Support for accessing Fastly's IPv6 ranges ([#189](https://github.com/fastly/go-fastly/pull/189)).
+- ip: Support for accessing Fastly's IPv6 ranges ([v#189](https://github.com/fastly/go-fastly/pull/189)).
```

Comment:
It seems there was a grep|sed error here as we now have a `v` character after every `[`. Similar comments for the rest of the file elided.

Reply:
https://github.com/fastly/go-fastly/pull/199/commits/d9b34a1ac761a9d8efcc122687b9cea7d7024ee5